### DISCUSSION TITLE
Logging in not working and saying users aren't found in the collection.

### DISCUSSION THREAD

**Discussion by beb050123** [pocketbase/pocketbase#2089] [Q&A]
Logging in does not work on my app. I'm not sure what it happening, because I have tested the values being passed to the login function and they match what is present in the "users" collection. Registering works fine, the data matches, everything is imported correctly, i'm using the correct base URL, and it isn't a problem with rules. Please help!

![image](https://user-images.githubusercontent.com/109166682/225987408-12612e2d-48e6-48b5-9dca-b98d9df9be11.png)
![image](https://user-images.githubusercontent.com/109166682/225987322-2b903448-3740-4c68-bad9-ff0388be63f5.png)

**Comment by ganigeorgiev** [comment]
You can try wrapping the `await` calls with try/catch and call `console.log(err, err.originalError)` to see if there are any errors.

Also does the `login` function needs to be a hook (_I assume that the above code is react_)?

If that doesn't help you pinpoint the issue, please try to create a minimal reproducible example with [stackblitz](stackblitz.com), [codesandbox](https://codesandbox.io/), or any other similar js code online env.
